plugins/database/src/db_plugin.cpp

```diff
@@ -11375,17 +11375,17 @@ irods::error db_move_object_op(
         len = strlen( collNameSlash );
         snprintf( collNameSlashLen, 10, "%d", len );
         cllBindVars[cllBindVarCount++] = newCollName;
-        cllBindVars[cllBindVarCount++] = ocLenStr;
+        cllBindVars[cllBindVarCount++] = oldCollName;
         cllBindVars[cllBindVarCount++] = newCollName;
-        cllBindVars[cllBindVarCount++] = ocLenStr;
-        cllBindVars[cllBindVarCount++] = collNameSlashLen;
```

Comment:
We might be able to remove `ocLenStr` and `collNameSlashLen` as they appear to only be used for this query.

Reply:
Good point, I have added a change to remove the unused declarations related to length calculation